src/ResourceManager/Automation/Commands.Automation/Microsoft.Azure.Commands.ResourceManager.Automation.dll-help.xml

```diff
@@ -14064,4 +14064,256 @@ PS C:\&gt; Start-AzureRmAutomationDscCompilationJob -ConfigurationName "Config01
     </maml:relatedLinks>
   </command:command>
 
+  <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
+    <command:details>
+      <command:name>Get-AzureRMAutomationHybridWorkerGroupGroup</command:name>
+      <maml:description>
+        <maml:para>Gets a Hybrid Runbook Worker Group</maml:para>
+      </maml:description>
+      <maml:copyright>
+        <maml:para />
+      </maml:copyright>
+      <command:verb>Get</command:verb>
+      <command:noun>AzureRMAutomationHybridWorkerGroup</command:noun>
+      <dev:version />
+    </command:details>
+    <maml:description>
+      <maml:para>
+        The Get-AzureRmAutomationHybridRunbookWorkerGroup cmdlet gets Azure Automation Hybrid runbook Worker Groups. To get a specific Group, specify its
+        name.
+      </maml:para>
+    </maml:description>
+    <command:syntax>
+      <command:syntaxItem>
+        <maml:name>Get-AzureRMAutomationHybridWorkerGroup</maml:name>
+        <command:parameter required="true" variableLength="false" globbing="false" pipelineInput="true (ByPropertyName)" position="0">
+          <maml:name>ResourceGroupName</maml:name>
+          <maml:description>
+            <maml:para> Specifies the name of the resource group for which this cmdlet gets hybrid runbook worker groups</maml:para>
+          </maml:description>
+          <command:parameterValue required="true" variableLength="false">String</command:parameterValue>
+        </command:parameter>
+        <command:parameter required="true" variableLength="false" globbing="false" pipelineInput="true (ByPropertyName)" position="1">
+          <maml:name>AutomationAccountName</maml:name>
+          <maml:description>
+            <maml:para>Specifies the name of an Automation account in which this cmdlet gets hybrid runbook worker group.</maml:para>
+          </maml:description>
+          <command:parameterValue required="true" variableLength="false">String</command:parameterValue>
+        </command:parameter>
+        <command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
```

Comment:
InformationAction and InformationVariable are common parameters in PowerShell 5.  No need to document.